components/style_traits/lib.rs

```diff
@@ -8,27 +8,36 @@
 
 #![crate_name = "style_traits"]
 #![crate_type = "rlib"]
-
 #![deny(unsafe_code, missing_docs)]
 
 extern crate app_units;
-#[macro_use] extern crate bitflags;
-#[macro_use] extern crate cssparser;
+#[macro_use]
+extern crate bitflags;
+#[macro_use]
+extern crate cssparser;
 extern crate euclid;
 extern crate malloc_size_of;
-#[macro_use] extern crate malloc_size_of_derive;
+#[macro_use]
+extern crate malloc_size_of_derive;
 extern crate selectors;
-#[cfg(feature = "servo")] #[macro_use] extern crate serde;
-#[cfg(feature = "servo")] extern crate webrender_api;
+#[cfg(feature = "servo")]
+#[macro_use]
+extern crate serde;
+#[cfg(feature = "servo")]
```

Comment:
This cfg(feature = "servo") needs to be moved above webrender_api instead of servo_arc.